src/pass.h

```diff
@@ -75,6 +75,7 @@ struct PassRunner {
 
   PassRunner(Module* wasm) : wasm(wasm), allocator(&wasm->allocator) {}
   PassRunner(Module* wasm, PassOptions options) : wasm(wasm), allocator(&wasm->allocator), options(options) {}
+  PassRunner(const PassRunner& that) = delete; // no copy constructor, we control |passes|
```

Comment:
You'll want to delete the copy assignment operator too.

Reply:
Thanks, done.